Source/VMware.vSphereDSC/VMware.vSphereDSC.Helper.psm1

```diff
@@ -192,4 +192,33 @@ function Update-PerfInterval
     )
 
     $PerformanceManager.UpdatePerfInterval($PerformanceInterval)
+}
+
+function Compare-Settings
```

Comment:
I think the function is good, but not applicable for the NTP use case, because it cannot cover all cases related to updating NTP Settings.

Reply:
Can you please specify which use cases are not met? 

The function is designed to be generic and reduce repeated code for resources, not to replace specific logic.

Reply:
Reset changes to VMware.vSphereDSC.psm1.